src/scripts/pokemons/PokemonFactory.ts

```diff
@@ -62,7 +62,7 @@ class PokemonFactory {
     public static routeDungeonTokens(route: number, region: GameConstants.Region): number {
         route = MapHelper.normalizeRoute(route, region);
 
-        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / 3, 1.05));
+        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / (3 / Math.round(1 + player.highestRegion() / 3)), 1.05));
```

Comment:
Do you think it would be better to just use the region we are passing instead of highest region?
```suggestion
        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / (3 / Math.round(1 + region / 3)), 1.05));
```

Reply:
My opinion : using highestRegion is better because at that point, you've already finished the entire dex, so the only reason to come back is for quests/achievements. Quests/achievements require a ton of runs, which cost quite a bit of Tokens. Making token gain based on the highestRegion makes the token farming less tedious